typescript-and-json/main.ts

```diff
@@ -0,0 +1,35 @@
+const books = [
+  {
+    isbn: '9780333791035',
+    title: 'The Great Gatsby',
+    author: 'F. Scott Fitzgerald',
+  },
+  {
+    isbn: '9780590353427',
+    title: 'Harry Potter',
+    author: 'J.K. Rowling',
+  },
+  {
+    isbn: '9780143303831',
+    title: 'Diary of A Wimpy Kid',
+    author: 'Jeff Kinney',
+  },
+];
+
+console.log('Array of Books:', books);
+console.log('type of books:', typeof books);
+
+const JSONbooks = JSON.stringify(books);
+
+console.log('JSON string of books:, JSONbooks');
+console.log('type of JSONbooks:', typeof JSONbooks);
+
```

Comment:
This is all one argument in your line 24 console.log so it is only logging the string `"JSON string of books:, JSONbooks"`